<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="floorPanelGroupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Floor Panel</string>
    </property>
    <widget class="QPushButton" name="floorPanelButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>input</string>
     </property>
    </widget>
    <widget class="QComboBox" name="floorPanelComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="floorPanelLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor</string>
     </property>
    </widget>
    <widget class="QLabel" name="floorPanelDestFloorLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dest. Floor</string>
     </property>
    </widget>
    <widget class="QComboBox" name="floorPanelDestFloorComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EmergencyButtonsGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Emergency buttons</string>
    </property>
    <widget class="QPushButton" name="EmergencyFireButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmergencyPowerOutageButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Outage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmergencyOverloadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmergencydoorBlockButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Door Block</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmergencyHelpButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SettingsGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QComboBox" name="SettingsFloorsComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="SettingsElevatorComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="SettingsFloorsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Floors</string>
     </property>
    </widget>
    <widget class="QLabel" name="SettingsElevatorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingsStartSimulationButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Simulation</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
